gymnasium/wrappers/render_collection.py

```diff
@@ -22,6 +24,10 @@ def __init__(self, env: gym.Env, pop_frames: bool = True, reset_clean: bool = Tr
         self.reset_clean = reset_clean
         self.pop_frames = pop_frames
 
+        self.metadata = copy.deepcopy(self.env.metadata)
+        if f"{self.env.render_mode}_list" not in self.metadata["render_modes"]:
```

Comment:
Technically, we allow all render modes to be list versions except "human"
Could you modify it to add all valid list render modes

Reply:
I think I originally had this implemented, but then I realized that's not exactly the case - when you use this wrapper, it takes the current environment's render mode and adds "_list" to it. And in principle, you can't change the `render_mode` of an environment after it's initialized. So for example, if you have an env with `render_mode="ansi"` and use the wrapper, there is no practical notion in which it now has access to `render_mode="rgb_array_list"` unless you reinitialize the whole thing

Which also makes me realize now that updating the metadata doesn't actually make that much of a difference now that `render` doesn't take any arguments.

Reply:
Good point, then I think this PR is good to merge